<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1024</width>
                <height>768</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>AnyEye v1.0</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="QFrame" name="menuFrame">
                        <property name="minimumSize">
                            <size>
                                <width>160</width>
                                <height>0</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>165</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="frameShape">
                            <enum>QFrame::Shape::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Shadow::Raised</enum>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayoutMenu">
                            <property name="spacing">
                                <number>10</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="logoLabel">
                                    <property name="styleSheet">
                                        <string notr="true">font-weight: bold; font-size: 18pt;</string>
                                    </property>
                                    <property name="text">
                                        <string>AnyEye</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignmentFlag::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnCamera">
                                    <property name="minimumSize">
                                        <size>
                                            <width>160</width>
                                            <height>0</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>üì∫ Cam√©ra</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnFaces">
                                    <property name="minimumSize">
                                        <size>
                                            <width>160</width>
                                            <height>0</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>üßë‚Äçü§ù‚Äçüßë Visages</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnHistory">
                                    <property name="minimumSize">
                                        <size>
                                            <width>160</width>
                                            <height>0</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>üïì Historique</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnSettings">
                                    <property name="minimumSize">
                                        <size>
                                            <width>160</width>
                                            <height>0</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>‚öôÔ∏è Param√®tres</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Orientation::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <layout class="QVBoxLayout" name="verticalLayoutContent">
                        <item>
                            <widget class="QStackedWidget" name="pagesWidget">
                                <widget class="QWidget" name="page"/>
                                <widget class="QWidget" name="page_2"/>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
